app/src/androidTest/java/com/github/se/icebreakrr/ui/profile/InboxProfileViewTest.kt

```diff
@@ -0,0 +1,161 @@
+package com.github.se.icebreakrr.ui.profile
+
+import androidx.compose.ui.test.assertHasClickAction
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsNotDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.test.espresso.Espresso.onIdle
```

Comment:
Question: useless import ?

Reply:
No, I use each one of these functions in my test.